Marlin/src/gcode/calibrate/G33.cpp

```diff
@@ -71,7 +71,9 @@ float lcd_probe_pt(const xy_pos_t &xy);
 
 void ac_home() {
   endstops.enable(true);
+  TERN_(SENSORLESS_PROBING, probe.set_homing_current(true));
```

Comment:
is not 
TERN_(HAS_HOMING_CURRENT , probe.set_homing_current(true));
?

Reply:
Hello.
In my opinion, it is correct. Only for SENSORLESS-PROBING enabled.

Reply:
I think this part is homing

Reply:
Yes, but it is part of the G33 self-calibration, therefore independent of SENSORLESS_HOMING since the SENSORLESS capacity depends on whether SENSORLESS_PROBING is enabled and not on SENSORLESS_HOMING. I don't see a need to unite them in G33, I think they were always independent in the code.

Reply:
Now that I think about it. It could be the case that the end stops were not eliminated, that homing would continue to depend on the physical switches, even if the probe had been eliminated.

Reply:
No but can use different pin. In my setup   I would try to use only sensorless probing

Reply:
I have modified it based on your recommendation. Now only if SENSORLESS_HOMING is enabled.